test/evergreen.yml

```diff
@@ -1593,6 +1593,65 @@ tasks:
           display_name: Coverage report
           remote_file: wiredtiger/${build_variant}/${revision}/coverage_report/coverage_report_${build_id}.html
 
+  # creating bunch of spinlock tests so they can be scheduled in parallel
+  - name: test-spinlock-gcc-1
```

Comment:
Can we name this spinlock-gcc-make-check-test? The convention is to put test in the end. Similar for the other tests.